pkg/webhooks/admission/configmap/validate/util.go

```diff
@@ -0,0 +1,43 @@
+package validate
```

Comment:
These functions should already exist in codebase？

Reply:
yes ， but it is in scheduler folder， so I repeat some

Reply:
yes ， but it is in scheduler folder， so I repeat some